src/auth/feedback-secret.guard.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  Injectable,
+  CanActivate,
+  ExecutionContext,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { Env } from 'src/env';
+
+@Injectable()
+export class FeedbackSecretGuard implements CanActivate {
+  constructor() {}
```

Comment:
PullProofAI

Consider adding a comment or documentation to describe the purpose of the `FeedbackSecretGuard` class.
```diff
+ // Guard to validate feedback secret for secure access
```


<!-- This is an auto-generated comment by PullProofAI -->